ruby/introduction/installing_ruby.md

```diff
@@ -287,6 +287,5 @@ Well done! Pat yourself on the back! The hard part is done, and it's time to mov
 
 #### Extras
 
-If you are using Visual Studio Code as your IDE, you can install the "Ruby LSP" extension which will provide you with semantic highlighting and formatting support. This is optional, but it is a quick install; go to the "Extensions" tab in VSC (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>), search "Ruby LSP", and click install on the first one. Congratulations, the extension is now installed (you can also uninstall the extension from here).
-
-If you are using a different IDE, a quick Google search such as "Ruby programming extensions for (your IDE here)" should provide you with the resources to get started. Free support extensions can help make your programming go more smoothly, and there are tons of extensions for all languages (not just Ruby).
+If you are using Visual Studio Code as your IDE, you can install the "Ruby LSP" extension which will provide you with semantic highlighting and formatting support. This is optional, but it is a quick install; go to the "Extensions" tab in VSC (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>), search "Ruby LSP", and click install on the first one. Congratulations, the extension is now installed (you can also uninstall the extension from here). Ruby LSP expects some configuration to be present that at this point would be unfeasible to introduce you to.
+You are going to get the most important features without the full configuration, nonetheless it is going to bug you about using a monorepo setup - you can choose "Don't show again" for the time being. It might also complain about not finding a lockfile or rubocop. Disregard these for now, they are going to be introduced later.

```

Comment:
I opted for "going to" since from what I've been taught it signifies that the thing in the future is rather certain or at least founded on something, while will is not certain. Is this a taste thing here or am I remembering my English classes wrong? 😆